src/BlackSlope.Api.Common/BlackSlope.Api.Common.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
     <LangVersion>7.2</LangVersion>

```

Comment:
Moving to C# 8.0 as part of the migration, thoughts?

Reply:
Yep I can make the change and see if everything works